scrapers/MindGeek.yml

```diff
@@ -0,0 +1,132 @@
+name: "MindGeek"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - 8thstreetlatinas.com/scene/
+      - babesnetwork.com/scene/
+      - bignaturals.com/scene/
+      - captainstabbin.com/scene/
+      - cumfiesta.com/scene/
+      - devianthardcore.com/scene/
+      - doghousedigital.com/
+      - dontbreakme.com/scene/
+      - fakehostel.com/scene/
+      - fakehub.com/scene/
+      - faketaxi.com/scene/
+      - familyhookups.com/scene/
+      - happytugs.com/scene/
+      - iconmale.com/
+      - iknowthatgirl.com/scene/
+      - letstryanal.com/scene/
+      - lilhumpers.com/scene/
+      - lookathernow.com/scene/
+      - milehighmedia.com/scene/
+      - milfhunter.com/scene/
+      - mofos.com/scene/
+      - mofosnetwork.com/scene/
+      - moneytalks.com/scene/
+      - publicagent.com/scene/
+      - publicpickups.com/scene/
+      - realityjunkies.com/scene/
+      - recklessinmiami.com/scene/
+      - rk.com/scene/
+      - roundandbrown.com/scene/
+      - seancody.com/scene/
+      - sneakysex.com/scene/
+      - sweetheartvideo.com/scene/
+      - sweetsinner.com/scene/
+      - teenslovehugecocks.com/scene/
+      - twistysnetwork.com/scene/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - 8thstreetlatinas.com/model/
+      - babesnetwork.com/model/
+      - bignaturals.com/model/
+      - captainstabbin.com/model/
+      - cumfiesta.com/model/
+      - devianthardcore.com/model/
+      - doghousedigital.com/
+      - dontbreakme.com/model/
+      - fakehostel.com/model/
```

Comment:
the fakehostel, fakehub, faketaxi sites should be removed from here as they don't have a model/performer section